.github/workflows/publish.yml

```diff
@@ -0,0 +1,89 @@
+name: Publish package to npmjs
+
+# This workflow runs when code is pushed to `main` (i.e: when a pull request is merged)
+on:
+  push:
+    branches: [main]
+
+jobs:
+    version:
+        runs-on: ubuntu-latest
+
+        # OSBotify will update the version on `main`, so this check is important to prevent an infinite loop
+        if: ${{ github.actor != 'OSBotify' }}
+
+        steps:
+            # Running this action ensures that only one instance of this job will run at a time.
+            # This is important to prevent race conditions when multiple pull requests are merged in quick succession
+            - uses: softprops/turnstyle@8db075d65b19bf94e6e8687b504db69938dc3c65
+              with:
+                  poll-interval-seconds: 10
+              env:
+                  GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+
+            - uses: actions/checkout@v3
+
+            - name: Decrypt & Import OSBotify GPG key
+              run: |
+                cd .github
+                gpg --quiet --batch --yes --decrypt --passphrase="$LARGE_SECRET_PASSPHRASE" --output OSBotify-private-key.asc OSBotify-private-key.asc.gpg
+                gpg --import OSBotify-private-key.asc
+              env:
+                LARGE_SECRET_PASSPHRASE: ${{ secrets.LARGE_SECRET_PASSPHRASE }}
```

Comment:
Just making sure this secret was added to the repo via Ring0?

Reply:
It wasn't, no. Is there anything you'd be able to point me to with instructions on what Ring0 needs to do exactly?

Reply:
Here is an example issue: https://github.com/Expensify/Expensify/issues/288197

`LARGE_SECRET_PASSPHRASE` is in 1Password, so Ring0 will be able to find it.

Reply:
OK, thanks. I opened this issue to have them added: https://github.com/Expensify/Expensify/issues/302806